src/pages/VolunteersCampaignWizard.page

```diff
@@ -15,107 +15,144 @@
+                    </div>
+                    <div class="slds-p-around--medium slds-theme--default">
+                        <apex:outputPanel id="panelJobsAndShifts" styleClass="slds-form--horizontal slds-grid">
+                            <div class="slds-col slds-size--1-of-2">
+                                <div class="slds-form-element">
+                                    <apex:outputLabel value="{!$Label.labelVolunteersWizardClone}" for="clonePicklist" styleClass="slds-form-element__label"/>
+                                    <apex:selectList value="{!campaignIdClone}" size="1" id="clonePicklist" styleClass="slds-form-element__control slds-select">
+                                        <apex:selectOptions value="{!listSOVolunteersCampaigns}"/>
+                                        <apex:actionSupport immediate="false" event="onchange" rerender="panelJobsAndShifts" status="statusLoad" action="{!ChangeCampaign}"/>
+                                    </apex:selectList>
+                                </div>
+                                <apex:outputPanel layout="block" rendered="{!campaignIdClone != null}" styleClass="slds-form-element">
+                                    <apex:outputLabel value="{!$Label.labelVolunteersWizardFirstShiftDate}" for="shiftStartDate" styleClass="slds-form-element__label"/>
+                                    <apex:outputField value="{!cmpShiftDates.StartDate}" id="shiftStartDate" styleClass="slds-form-element__control slds-form-element__static"/>
+                                </apex:outputPanel>
+                                <apex:outputPanel layout="block" rendered="{!campaignIdClone != null}" styleClass="slds-form-element">
+                                    <apex:outputLabel value="{!$Label.labelVolunteersWizardNewShiftDate}" for="shiftEndDate" styleClass="slds-form-element__label"/>
+                                    <apex:inputField value="{!cmpShiftDates.EndDate}" id="shiftEndDate" styleClass="slds-form-element__control slds-input dateOnlyInput"/>
+                                </apex:outputPanel>
+                                <apex:outputPanel layout="block" rendered="{!campaignIdClone != null}" styleClass="slds-form-element">
+                                    <apex:outputLabel value="{!$Label.labelVolunteersWizardCopyVolunteers}" for="copyVolunteers" styleClass="slds-form-element__label"/>
+                                    <apex:inputCheckbox value="{!fCopyVolunteers}" id="copyVolunteers" styleClass="slds-form-element__control"/>
+                                </apex:outputPanel>
+                                <apex:outputPanel layout="block" rendered="{!campaignIdClone != null}" styleClass="slds-form-element">
+                                    <apex:outputLabel value="{!$Label.labelVolunteersWizardOverrideHoursStatus}" styleClass="slds-form-element__label" for="hoursStatus"/>
```

Comment:
the weird layout occurs when the page width isn't really large.  so if you don't see it, try playing with your window width.